packages/types/src/documents/app/table.ts

```diff
@@ -69,7 +69,7 @@ export interface Table extends Document {
   constrained?: string[]
   sql?: boolean
   indexes?: { [key: string]: any }
-  dataImport?: { [key: string]: any }
```

Comment:
Just had a look through here - noticed the type change, if this means that the API has changed from accepting the `dataImport` property, then might need to update the API doc comments in the `routes/table.ts` file, theres still a few mentions of it there!

Reply:
Ah yes sorry I was meant to ask someone about that, also, I notice there were comments above each route on the server describing the payload etc, are those written by hand or generated?

Reply:
Those were written by hand, as part of plans for an APIDoc/OpenAPI generator for the internal API, although the docs are barely finished, its just useful for the table/rows APIs since they are so core to the whole system!